diff --git a/waybar/.config/waybar/config b/waybar/.config/waybar/config
index 4ccdabf..f9d263a 100644
--- a/waybar/.config/waybar/config
+++ b/waybar/.config/waybar/config
@@ -18,6 +18,8 @@
         "memory",
         "disk#home",
         "disk#root",
+        "battery",
+        "backlight",
         "pulseaudio",
         "bluetooth",
         "network",
@@ -89,7 +91,7 @@
         "tooltip": false
     },
     "temperature#cpu": {
-        "hwmon-path": "/sys/class/hwmon/hwmon2/temp1_input", // check if path is correct for CPU temps
+        "hwmon-path": "/sys/class/hwmon/hwmon4/temp1_input", // check if path is correct for CPU temps
         "critical-threshold": 80,
         "format-critical": "{icon} CPU: {temperatureC}°C",
         "format": "{icon} CPU: {temperatureC}°C",
@@ -135,6 +137,7 @@
             "default": ["", ""]
         },
         "scroll-step": 1,
+        "smooth-scrolling-threshold": 2,
         "on-click": "pavucontrol"
     },
     "network": {
@@ -178,7 +181,8 @@
         "format-plugged": " {icon} {capacity}%",
         "on-click": "", // update module on click
         "signal": 10, // for udev rule to force update the module when it doesn't do so itself
-        "max-length": 25
+        "max-length": 25,
+        "on-update": "$HOME/.config/waybar/low_battery_warning.sh"
     },
     "backlight": {
         "device": "intel_backlight",
diff --git a/zsh/.zprofile b/zsh/.zprofile
index 1a8cf23..f96b048 100644
--- a/zsh/.zprofile
+++ b/zsh/.zprofile
@@ -1,5 +1,5 @@
 # used to differentiate laptop/desktop configs
-export LAPTOP_CONFIG=false
+export LAPTOP_CONFIG=true
 
 # default programs
 export EDITOR="nvim"
